packages/dds/map/package.json

```diff
@@ -60,7 +60,7 @@
     "@fluidframework/core-interfaces": "^0.39.0",
     "@fluidframework/datastore-definitions": "^0.40.0",
     "@fluidframework/driver-utils": "^0.40.0",
-    "@fluidframework/protocol-base": "^0.1025.0-0",
+    "@fluidframework/protocol-base": "0.1025.0-26003",
```

Comment:
released packages shouldn't depend on patch builds. you shouldn't need any of these changes, as once 1025.* is released, we just need to bump the lerna lock in client to use it